src/cli/cli.cpp

```diff
@@ -7808,6 +7839,18 @@ template <> otError Interpreter::Process<Cmd("trel")>(Arg aArgs[])
 exit:
     return error;
 }
+
+void Interpreter::OutputTrelCounters(const otTrelCounters &aCounters)
+{
+    Uint64StringBuffer u64StringBuffer;
+
+    OutputLine("Inbound: Packets %s Bytes %s", Uint64ToString(aCounters.mRxPackets, u64StringBuffer),
+               Uint64ToString(aCounters.mRxBytes, u64StringBuffer));
```

Comment:
We are using the same `u64StringBuffer` in multiple calls to `Uint64ToString()` within the same statement (all as inputs to `OutputLin()`). This will not work correctly and the buffer will be overwritten.

We should either use multiple  buffers, or do this in multiple `Output()` calls. Output values one by one in their own `OutputFormat()` statemenst (would suggest this way).

Reply:
Done.